src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1025,6 +1027,10 @@ public void onRemoteOperationFinish(RemoteOperation operation, RemoteOperationRe
             onGetUserNameFinish(result);
         }
 
+        if (result.isSuccess() && mAction == ACTION_UPDATE_EXPIRED_TOKEN) {
+            setResult(FileActivity.RESULT_OK);

```

Comment:
Of course, we can use Activity.RESULT_OK but this change involves adding the import android.app.Activity too.